src/main/java/xonin/backhand/CommonProxy.java

```diff
@@ -1,23 +1,25 @@
 package xonin.backhand;
 
-import javax.annotation.Nullable;
-
-import net.minecraft.entity.Entity;
 import net.minecraft.entity.player.EntityPlayer;
 import net.minecraft.entity.player.EntityPlayerMP;
 import net.minecraft.item.ItemStack;
-import net.minecraftforge.common.util.FakePlayer;
 
 import com.gtnewhorizon.gtnhlib.eventbus.EventBusSubscriber;
+import com.gtnewhorizon.gtnhlib.keybind.SyncedKeybind;
 
 import cpw.mods.fml.common.eventhandler.SubscribeEvent;
 import cpw.mods.fml.common.gameevent.PlayerEvent;
 import xonin.backhand.api.core.BackhandUtils;
+import xonin.backhand.packet.BackhandPacketHandler;
 import xonin.backhand.packet.OffhandSyncItemPacket;
 
 @EventBusSubscriber
 public class CommonProxy {
 
+    public static final SyncedKeybind SWAP_KEY = SyncedKeybind
+        .createConfigurable("backhand.swap_offhand", "key.categories.gameplay", 33)
```

Comment:
LWJGL `Keyboard` class should be safe to still use here on the server so that this key code can be displayed a bit more clearly